### ISSUE TITLE
Page Size doesnt change anything

### ISSUE THREAD

**Issue by StephaneSeguin** [lucasalexander/AlexanderDevelopment.ProcessRunner#3]
Hello, I'm changing the Page size to limit the number of record by batch created.
We have more then 5000 records to update. 

I'm getting a 2 min timeout error.  Is the page size should overcome the 2 min timeout?

Also unable to use top =1000 in the fetch query because Page Size parameter is used.

**Comment by lucasalexander** [comment]
No, changing the page size wouldn't be expected to resolve the issue. My recommendation is to split the data into multiple batches. For example, if you have a recurring process to execute on contacts, you could add an additional filter criterion for last names starting with a particular set of letters. You could then clone that recurring process to select contacts with last names starting with a different set of letters. This approach should let you work around the timeout problem.
